db/migrate/20150629201312_create_beaches.rb

```diff
@@ -0,0 +1,17 @@
+class CreateBeaches < ActiveRecord::Migration
+  def change
+    create_table :beaches do |t|
+      t.string :street1, null: false
+      t.string :street2, null: false
+      t.string :city, null: false
+      t.string :state, null: false
+      t.string :zip, null: false
+      t.string :name, null: false
+      t.integer :entrance_fee
```

Comment:
will you express entrance fees as the price in cents? If not, you should use use a `double` datatype with appropriate scale and precision